### PR TITLE
Update docs for offline install with postgres.

### PR DESCRIPTION
**Pull request by myersCody**
Update docs for offline install with postgres.

### CODE REVIEW

File: docs/source/install_container_image.rst [hunk 1]
```diff
@@ -26,3 +26,18 @@ The output appears similar to the following example::
...
+
+How to Create a Postgres Image
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. After Docker is installed, you can obtain the postgres container image with these steps::
+
+    docker pull postgres:9.6.10
+    docker save -o postgres.9.6.10.tar postgres:9.6.10
+
+**Note:** These steps do require the use of the internet.
```
Comment (abaiken) [inline docs/source/install_container_image.rst]: I would change this to say `These steps do require internet connectivity.`

File: docs/source/install_container_image.rst [hunk 2]
```diff
@@ -26,3 +26,18 @@ The output appears similar to the following example::
...
   REPOSITORY              TAG                 IMAGE ID            CREATED             SIZE
   quipucords              0.0.46               fdadcc4b326f        3 days ago          969MB
+
+
+.. _postgres-image-create:
+
+How to Create a Postgres Image
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. After Docker is installed, you can obtain the postgres container image with these steps::
```
Comment (abaiken) [inline docs/source/install_container_image.rst]: change `these steps` to `the following steps`

File: docs/source/install_container_image.rst [hunk 3]
```diff
@@ -26,3 +26,18 @@ The output appears similar to the following example::
 
   REPOSITORY              TAG                 IMAGE ID            CREATED             SIZE
   quipucords              0.0.46               fdadcc4b326f        3 days ago          969MB
+
+
+.. _postgres-image-create:
+
+How to Create a Postgres Image
```
Comment (abaiken) [inline docs/source/install_container_image.rst]: I would change the name to `Building the Postgres Image`

**Comment by kholdaway** [comment]
Regarding this section:
- https://quipucords.readthedocs.io/en/postgres_doc_update/install.html#installing-without-internet-connectivity

Let's clearly distinguish between what requires connectivity to the internet and what does not.  Additionally, we should clearly denote when we are operating on the target machine for the QPC install verse gathering packages (e.g. building postgres)

Sections could be ordered as follows:
1) Gathering software packages required for offline installation (this step requires connectivity).  These steps are on a machine that will not host the QPC server.
2) Installing pre-requisites on machine you intend to install the QPC server on.  Internet connectivity may be limited.
3) Offline installation steps for machine described in 2

File: docs/source/install_scripted.rst [hunk 4]
```diff
@@ -70,6 +61,9 @@ After the dependencies are installed, you must gather packages through the `Gith
 - Fedora 27 (`Download <https://github.com/quipucords/quipucords/releases/download/0.0.46/qpc-0.0.46-ACTUAL_COPR_GIT_COMMIT.fc27.noarch.rpm>`_)
 - Fedora 28 (`Download <https://github.com/quipucords/quipucords/releases/download/0.0.46/qpc-0.0.46-ACTUAL_COPR_GIT_COMMIT.fc28.noarch.rpm>`_)
 
+**Transfer Packages**
+
+After the required packages have been collected, they will need to be transfered to the machine without internet connectivity.
```
Comment (abaiken) [inline docs/source/install_scripted.rst]: I think that `transfered` has two rs (transferred)

File: docs/source/install_scripted.rst [hunk 5]
```diff
@@ -83,7 +77,25 @@ After the dependencies are installed, you must gather packages through the `Gith
...
+
+**Command Line Tool Dependencies:**
+
+- `Download & Configure EPEL <install.html#commandline>`_
+- Python 3.4
+- Python 3.4-requests
+
+**Start Offline Install**
+
+Start the offline installtiion by entering the following command. Alternatively, enter the following command with options as described in `Installation Options`_::
```
Comment (abaiken) [inline docs/source/install_scripted.rst]: change `installtiion` to `installation`

File: docs/source/install_scripted.rst [hunk 6]
```diff
@@ -38,30 +38,21 @@ The output appears similar to the following example::
...
-- Python 3.4-requests
+- Server Container Image (`Download <https://github.com/quipucords/quipucords/releases/download/0.0.46/quipucords.0.0.46.tar.gz>`_)
 
-Packages
-++++++++
-After the dependencies are installed, you must gather packages through the `Github Release Page <https://github.com/quipucords/quipucords/releases/tag/0.0.46>`_.
+**Build the Postgres Image:**
 
-**Server Packages:**
+- Install Docker (`Documentation <https://docs.docker.com/install/>`_)
```
Comment (kholdaway) [inline docs/source/install_scripted.rst]: Let's put a little introduction here.

```
The Quipucords server requires a Postgres Docker image.  You must build the image on a machine with internet connectivity.  Follow the steps documented below to create a Postgres Docker image.
```

Ashley will probably correct that more, but that is the general idea.

File: docs/source/install_scripted.rst [hunk 7]
```diff
@@ -70,6 +61,9 @@ After the dependencies are installed, you must gather packages through the `Gith
 - Fedora 27 (`Download <https://github.com/quipucords/quipucords/releases/download/0.0.46/qpc-0.0.46-ACTUAL_COPR_GIT_COMMIT.fc27.noarch.rpm>`_)
 - Fedora 28 (`Download <https://github.com/quipucords/quipucords/releases/download/0.0.46/qpc-0.0.46-ACTUAL_COPR_GIT_COMMIT.fc28.noarch.rpm>`_)
 
+**Transfer Packages**
+
+After the required packages have been collected, they will need to be transferred to the machine without internet connectivity.
```
Comment (kholdaway) [inline docs/source/install_scripted.rst]: I would say
```After the required packages have been collected, they will need to be transferred to the machine where the Quipucords server will be installed.```

File: docs/source/install_scripted.rst [hunk 8]
```diff
@@ -38,30 +38,23 @@ The output appears similar to the following example::
 
 Installing without Internet Connectivity
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-If you choose the offline option to run the installer, the following associated dependencies must be installed.
+If you choose the offline option to run the installer, you will need to obtain the following packages on a machine with internet connectivity.
 
-**Server Dependencies:**
+**Server Package:**
```
Comment (kholdaway) [inline docs/source/install_scripted.rst]: `Quipucords Server Package`

File: docs/source/install_scripted.rst [hunk 9]
```diff
@@ -38,30 +38,23 @@ The output appears similar to the following example::
...
 
-- `Ansible <install.html#installing-the-ansible-prerequisite>`_
-- `Docker <install.html#installing-docker-and-the-quipucords-server-container-image>`_
+- Server Container Image (`Download <https://github.com/quipucords/quipucords/releases/download/0.0.46/quipucords.0.0.46.tar.gz>`_)
 
-**Command Line Tool Dependencies:**
+**Build the Postgres Image:**
```
Comment (kholdaway) [inline docs/source/install_scripted.rst]: `Build the Postgres Docker Image`

File: docs/source/install_scripted.rst [hunk 10]
```diff
@@ -38,30 +38,23 @@ The output appears similar to the following example::
...
 
-- `Download & Configure EPEL <install.html#commandline>`_
-- Python 3.4
-- Python 3.4-requests
-
-Packages
-++++++++
-After the dependencies are installed, you must gather packages through the `Github Release Page <https://github.com/quipucords/quipucords/releases/tag/0.0.46>`_.
+The Quipucords server requires a Postgres Docker image.  You must build the image on a machine with internet connectivity.  Follow the steps documented below to create a Postgres Docker image.
```
Comment (kholdaway) [inline docs/source/install_scripted.rst]:
```
The Quipucords server requires a Postgres Docker image.  You must build the Docker image on a machine with internet connectivity.  Follow the steps documented below to create a Postgres Docker image.
```

**Pull request merged**